name.abuchen.portfolio/src/name/abuchen/portfolio/online/Column.java

```diff
@@ -0,0 +1,166 @@
+package name.abuchen.portfolio.online;
```

Comment:
Is there a specific reason to move "Column" to another package than the only class that is using it? I would prefer to have it next to HTMLTableQuoteFeed (at least as long it is bound very close to HTMLTableQuoteFeed).